### ISSUE TITLE
Impossible to navigate away from 404s?

### ISSUE THREAD

**Issue by daniel-ac-martin** [daniel-ac-martin/NotGovUK#282]
We've had a report that it is impossible to navigate away from a 404 page when in client-side rendering.

The only way to test this would be to deliberately create a broken link on a site.

**Comment by RobinKnipe** [comment]
Surely you don't need to "create" a broken link, just try and hit a page that doesn't exist, no?

**Comment by daniel-ac-martin** [comment]
No, because that would cause the browser to make a request against the server, which works fine.

Instead, we need react-router to intercept the click and try to route the 'request' in the client.
